DependencyInjection/Configuration.php

```diff
@@ -167,4 +168,27 @@ private function getOrganizeMigrationsModes() : array
 
         return $namesArray;
     }
+
+    /**
+     * Returns the correct deprecation param's as an array for setDeprecated.
+     *
+     * Symfony/Config v5.1 introduces a deprecation notice when calling
```

Comment:
Package names are not capitalized

```suggestion
     * symfony/config v5.1 introduces a deprecation notice when calling
```